java/src/jmri/jmrit/display/DisplayBundle.properties

```diff
@@ -384,6 +384,9 @@ PanelStoreIncomplete = The storing of your panel is incomplete and may result in
 StoreError = Error Storing Information!
 StoreHasErrors = Errors experienced during store.
 StoreIncomplete = The storing of your information is incomplete and may result in missing items
+FileOverwriteWarning1 = File
```

Comment:
FileOverwriteWarning1 and FileOverwriteWarning2 should be one string: "<html>File {} already exists.<br>Overwrite it?</html>"